src/Annotations/Command/QueueWatchCommand.php

```diff
@@ -72,8 +72,8 @@ protected function process(InputInterface $input, QueueItem $item, $entity = nul
                 $upsert = $this->hypothesisSdk->users()->upsert($user)->wait();
                 $this->logger->info(sprintf('Hypothesis user "%s" successfully %s.', $upsert->getUsername(), ($upsert->isNew() ? 'created' : 'updated')));
             } catch (BadResponse $e) {
-                // If upsert fails, log error but don't repeat.
-                if (in_array($e->getResponse()->getStatusCode(), [Response::HTTP_BAD_REQUEST, Response::HTTP_NOT_FOUND])) {
+                // If client error detected, log error and don't repeat.
+                if (in_array($e->getResponse()->getStatusCode(), range(Response::HTTP_BAD_REQUEST, 499))) {
```

Comment:
Rather verbose. `$e->getResponse()->getStatusCode() < 500` would be fine.

Reply:
Done.